packages/progress-circle/src/ProgressCircle.ts

```diff
@@ -83,6 +93,9 @@ export class ProgressCircle extends SizedMixin(SpectrumElement, {
         ];
         const masks = ['Mask1', 'Mask2'];
         return html`
+            <div style="display:none">
+                <slot @slotchange=${this.onSlotChange}></slot>
+            </div>
```

Comment:
Interesting approach. You can `display: none` the `slot` rather than providing extra wrapping DOM. However, all styling should be applied via the associated CSS files when possible.